src/storage/txn/actions/get_old_value.rs

```diff
@@ -214,9 +220,16 @@ mod tests {
                     .reader
                     .seek_write(&Key::from_raw(b"a"), case.written.last().unwrap().1)
                     .unwrap()
-                    .unwrap()
-                    .1;
-                let result = get_old_value(&mut txn, &Key::from_raw(b"a"), prev_write).unwrap();
+                    .map(|w| w.1);
+                let prev_write_loaded = true;
+                let result = get_old_value(
+                    &mut txn,
+                    &Key::from_raw(b"a"),
+                    TimeStamp::new(25),
+                    prev_write_loaded,
+                    prev_write,
+                )
+                .unwrap();
```

Comment:
There's a test case in master that checks prev_write_loaded is true and prev_write is none. It seems that case is missing here?